### PR TITLE
Land/3 percept updated to land/5 percept

### PR DESCRIPTION
**Pull request by StefanV7**
It is now land(`<LandID>`,`<Owner>`,`<MultiPolygon>`,`<ZoneList>`,`<Area>`). #82

### CODE REVIEW

**Comment by jannelie** [comment]
can I see the travis build?

**Comment by MaxGroenenboom** [comment]
Here it is: https://travis-ci.org/levilime/tygron/builds/136077580 (and it fails btw)

Oh but it's only 2 checkstyle issues:
`[ERROR] src/main/java/contextvh/ContextEntityEventHandler.java:[108] (sizes) LineLength: Line is longer than 120 characters (found 124).
[ERROR] src/main/java/contextvh/ContextEntityEventHandler.java:[108,77] (whitespace) GenericWhitespace: '>' is followed by whitespace.`

**Comment by jannelie** [comment]
you should fix this first

**Comment by nskartoredjo** [comment]
The same discussion from #78 applies to this PR.

Also I like to see `Land b` to be changed to something more understandable (like in #78).

**Comment by StefanV7** [comment]
I will fix the checkstyle errors.

Currently the MultiPolygon is already written as a string.
We don't plan on parsing it, and currently use it as a string, so that will remain as it is now. That issue is beyond the scope of this pull request.

**Comment by StefanV7** [comment]
The checkstyle erros have been fixed.

**Comment by StefanV7** [comment]
Uhmm... we know there are merge conflicts right now, wait a sec.

File: contextvh/src/test/java/contextvh/TestEnvironment.java [hunk 1]
```diff
@@ -13,11 +14,24 @@
 import org.junit.Test;
 
 import eis.EnvironmentListener;
+=======
```
Comment (spiderbiggen) [inline contextvh/src/test/java/contextvh/TestEnvironment.java]: you left the merge conflict lines in

**Comment by MaxGroenenboom** [comment]
Can you revert the changes in the sdk-tests directory? Those could cause conflicts at a later date.

**Comment by StefanV7** [comment]
Yeah, I am fixing everything right now. I know it went wrong.

**Pull request closed without merging**